src/main/sensors/battery.c

```diff
@@ -108,8 +108,8 @@ uint16_t batteryAdcToVoltage(uint16_t src)
 
 int32_t currentSensorToCentiamps(uint16_t src)
 {
-    int32_t millivolts = ((uint32_t)src * ADCVREF) / 0xFFF - batteryConfig()->current.offset;
-    return millivolts * 1000 / batteryConfig()->current.scale; // current in 0.01A steps
+    int32_t decimillivolts = ((uint32_t)src * ADCVREF * 10) / 0xFFF - batteryConfig()->current.offset * 10;
+    return decimillivolts * 100 / batteryConfig()->current.scale; // current in 0.01A steps
```

Comment:
Why not calculate in microvolts and get rid of `* 100` here?

Reply:
Indeed, why not. I'd like to change the offset voltage unit too. With scale = 100, 1mV = 10mA. 0.1mV steps would be nicer. Plus since the scale is already 0.1mV steps it would match.

Reply:
Let's not change the offset units in a minor release. I want to include this into `1.9.1`

Reply:
OK. Done.